### PR TITLE
Op (FusedEmbeddingSeqPool) error message enhancement.

### PR DESCRIPTION
**Pull request by lelelelelez**
### 不合规报错检查增强

[测试脚本](https://github.com/lelelelelez/OtherProject/blob/master/test_fusedembeddingseqpoolop.py)
**1. Op InferShape的输入输出检查**

- 修改前报错：
  `Input W of FusedEmbeddingSeqPoolOp should not be null.`

- 修改后使用新的检测宏`OP_INOUT_CHECK(ctx->HasInput("X"), "Input", "X", "Mul")`，保证Op的输入输出检查报错信息均一致。
  `NotFoundError: No Input(Ids) found for FusedEmbeddingSeqPool operator.`

**2. 输入的table的dim等于2**

- 修改前无报错

- 修改后报错

  `InvalidArgumentError: The dim size of the input tensor 'W' should be 2. But received W's size =1 `

**3. Input tensor Ids's dim size 大于等于1**

- 修改前报错：

  `The dim size of the 'Ids' tensor must greater than 1.`

- 修改后报错：

  `InvalidArgumentError: The dim size of the 'Ids' tensor should be greater than or equal to 1. But received Id's size = 0`

**4. The last dimension of the input tensor 'Ids' 等于1**

- 修改前报错：

  `The last dimension of the 'Ids' tensor must be 1.`

- 修改后报错：

  `InvalidArgumentError: The last dimension of the input tensor 'Ids' should be 1. But received Ids's size in the last dimension = 2`

**5. combiner 目前只支持sum**

- 修改前无报错

- 修改后报错：

  `Unimplemented: The pooling type of sequence_pool only support sum now. So the 'combiner' must be 'sum'.`

**6. the LoD Level of Ids等于1**

- 修改前无报错

- 修改后报错

  `InvalidArgumentError: In compile time, the LoD Level of Ids should be 1. But received the LoD Level of Ids = 2`

**7. table_width * idx_width 小于等于 out_width**

- 修改前无报错

- 修改后报错：

  `InvalidArgumentError: table_width * idx_width should be less than or equal to out_width. But received table_width * idx_width = %s, out_width = %d. `

**8. Ids's lod[0] 大于0**

- 修改前报错：

  `The LoD[0] could NOT be empty`

- 修改后报错：

  `InvalidArgumentError: The LoD[0] should be greater than 1. But received LoD[0] = 0`

**9. the LoD of ids 等于 1**

- 修改前报错：

  `The LoD level of Input(Ids) must be 1`

- 修改后报错：

  `InvalidArgumentError: The LoD level of Input(Ids) should be 1. But received Ids's LoD level = 0`

### CODE REVIEW

File: paddle/fluid/operators/fused/fused_embedding_seq_pool_op.cc [hunk 1]
```diff
@@ -24,30 +24,47 @@ class FusedEmbeddingSeqPoolOp : public framework::OperatorWithKernel {
...
+                      platform::errors::InvalidArgument(
+                          "The dim size of the input tensor 'W' should be 2. "
+                          "But received W's size = %d.",
+                          table_dims.size()));
+    PADDLE_ENFORCE_GE(
+        ids_dims.size(), 1,
+        platform::errors::InvalidArgument(
+            "The dim size of the input tensor 'Ids' should be greater "
+            "than or equal to 1. But received Ids's size = %d.",
+            ids_dims.size()));
```
Comment (luotao1) [inline paddle/fluid/operators/fused/fused_embedding_seq_pool_op.cc:46]: 请问这行判断是否可以去掉，会有shape等于0的情况么？
@Aurelius84

File: paddle/fluid/operators/fused/fused_embedding_seq_pool_op.h [hunk 2]
```diff
@@ -90,8 +90,17 @@ struct EmbeddingVSumFunctor {
...
+                      platform::errors::InvalidArgument(
+                          "table_width * idx_width should be less than or "
+                          "equal to out_width. But received "
+                          "table_width * idx_width = %s, out_width = %d.",
+                          table_width * idx_width, out_width));
+    PADDLE_ENFORCE_GT(ids_lod.size(), 1UL,
+                      platform::errors::InvalidArgument(
+                          "The tensor ids's LoD[0] should be greater than 1. "
+                          "But received the ids's LoD[0] = %d.",
+                          ids_lod.size()));
```
Comment (luotao1) [inline paddle/fluid/operators/fused/fused_embedding_seq_pool_op.h:103]: 这里报错应该和143-145行的错误信息一致。

File: paddle/fluid/operators/fused/fused_embedding_seq_pool_op.cc (hunk 1 above)
Comment (Aurelius84) [reply paddle/fluid/operators/fused/fused_embedding_seq_pool_op.cc:46]: 应该可以去掉。

Comment (luotao1) [reply paddle/fluid/operators/fused/fused_embedding_seq_pool_op.cc:46]: @lelelelelez 可以考虑下一个PR去掉

**Review by luotao1 — APPROVED** [review]
LGTM

**Pull request merged**